public/javascripts/SVLabel/src/SVLabel/Map.js

```diff
@@ -486,6 +486,19 @@ function Map ($, google, turf, params) {
         if ('compass' in svl) { svl.compass.update(); }
     }
 
+    // missions greater than 3000 feet are measured in miles
+    function distanceLeftFeetOrMiles(){
+        var completeRate = 1 - svl.missionContainer.getCurrentMission().getMissionCompletionRate();
```

Comment:
Can you change the variable name to `remainingRate` rather than `completeRate`? I think that's what it means?